<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FotoYa</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="pbOffline">
     <property name="text">
      <string>Ingresar off-line</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pbIngresar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbConfiguracion">
     <property name="text">
      <string>Configuraci√≥n</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbFacebook">
     <property name="title">
      <string>Facebook</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lUsuarioFaceebook">
        <property name="text">
         <string>Usuario</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lClaveFacebook">
        <property name="text">
         <string>Clave</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leClaveFacebook">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>f0t0y400</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leUsuarioFacebook">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>juangabrielaguilera@outlook.com</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="cbWebFacebook">
          <property name="text">
           <string>Facebook Web</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lInfoFacebook">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="gbTwitter">
     <property name="title">
      <string>Twitter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lUsuario">
        <property name="text">
         <string>Usuario</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leUsuarioTwitter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>juangabrielaguilera@outlook.com</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leClaveTwitter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>f0t0y400</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lClave">
        <property name="text">
         <string>Clave</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cbWebTwitter">
          <property name="text">
           <string>Twitter Web</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lInfoTwitter">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
